module-7/assignments/assignment.js

```diff
@@ -0,0 +1,39 @@
+data = [
+    { born: 1870, died: 1924 },
+    { born: 1893, died: 1976 },
+    { born: 1869, died: 1948 },
+    { born: 1901, died: 1989 },
+];
+
+const age = data.map((entry) => {
+    return entry.died - entry.born;
+});
+
+console.log(age);
+
+const filtered = age.filter((a) => {
+    return a > 75;
+});
+
+console.log(filtered);
+
+const oldest = filtered.reduce((max, age) => {
+    if (age > max) {
+        max = age;
+    }
+    return max;
+}, -1);
```

Comment:
no need to specify initial value here. it will be converted into number `0` when you do >
```suggestion
const oldest = filtered.reduce((max, age) => {
    if (age > max) {
        return age;
    }
    
    return max;
});
```